landcover.mss

```diff
@@ -689,18 +710,151 @@
   }
 }
 
+@ridge-text:          #7A2F18;
+@valley-text:         #305040;
+@mountain_range-text: #C03A01;
+@massif-text:         #C03A01;
+@cliff-text:          #999;
+@arete-text:          #800000;
+@couloir-text:        #0080C0;
+@dale-text:           #008000;
+@gorge-text:          #800040;
```

Comment:
It feels like there might be too many colours here - some of them should probably be grouped under a common name